package.json

```diff
@@ -47,6 +47,7 @@
 		"@nextcloud/browserslist-config": "^2.2.0",
 		"@nextcloud/eslint-config": "^6.1.0",
 		"@nextcloud/stylelint-config": "^1.0.0-beta.0",
-		"@nextcloud/webpack-vue-config": "^4.1.0"
+		"@nextcloud/webpack-vue-config": "4.0.3",
+		"webpack": "5.38.1"
```

Comment:
Not needed with `@nextcloud/webpack-vue-config`

Reply:
Webpack > `5.48.0` apparently break source map generation, investigating